tests/testRingsSkyMap.py

```diff
@@ -26,6 +26,17 @@ def setUp(self):
     def testTractSeparation(self):
         self.skipTest("A particular tract separation is not important for RingsSkyMap")
 
+    def testPoles(self):
+        """Test that findAllTracts behaves at the poles
+
+        Testing fix to DM-10686.
+        """
+        skymap = self.getSkyMap()
+        deg = lsst.afw.geom.degrees
+        tracts = skymap.findAllTracts(lsst.afw.coord.IcrsCoord(321*deg, 90*deg))
```

Comment:
Since this is an important edge case, it might be good to do the tests on multiple different RAs.